content/5.resources/2.ibc-channels.md

```diff
@@ -14,10 +15,8 @@ Here, you can find a comprehensive list of channels that connect the Archway blo
 
 The chain ID for the Constantine testnet is `constantine-3`.
 
-| **Source Channel** | **Destination** | **Destination Chain ID**     | **Destination Channel** |
-| :----------------- |:----------------|------------------------------|-------------------------|
-| channel-4          | Osmosis         | osmo-test-5                  | channel-120              |
-| channel-8          | Axelar          | axelar-testnet-lisbon-3      | channel-259             |
-| channel-7          | Cosmos Hub      | theta-testnet-001            | channel-2748            |
+::relayersTable{:repoFiles="testnetRelayerFiles"}
+::
+<!-- <relayers-table :repo-files="testnetRelayerFiles" /> -->
```

Comment:
I discovered that it is also possible to use the component like this (line 20), so line 18 and line 20 are two different syntax to use the same component in markdown